lib/browser/guest-window-manager.js

```diff
@@ -78,19 +80,8 @@ const setupGuest = function (embedder, frameName, guest, options) {
     embedder.send('ELECTRON_GUEST_WINDOW_MANAGER_WINDOW_CLOSED_' + guestId)
     embedder.removeListener('render-view-deleted', closedByEmbedder)
   }
-  if (!options.webPreferences.sandbox) {
-    // These events should only be handled when the guest window is opened by a
-    // non-sandboxed renderer for two reasons:
```

Comment:
@tarruda I removed this because I was seeing a spec failure once `options.webPreferences.sandbox` was inherited by default from the parent window (line 16).

It looks like it was going down this path previously when a sandboxed window was opening a new window so I wasn't sure if this code was still needed.

Is this check only needed for opening sandboxed windows opened from non-sandbox windows?

On master, if you open a sandbox window then do `window.open()` the listeners did appear to be registered since the options have no web preferences by default so `options.webPreferences.sandbox` is `false`.